sites/officer.com/server/templates/index.marko

```diff
@@ -116,6 +116,24 @@ $ const { id, alias, name, pageNode } = data;
       </div>
     </div>
 
+    <div class="row">
+      <div class="col-lg-12 mb-block">
+        <div class="node-list node-list--inner-justified">
+          <div class="node-list__header">
```

Comment:
Use the node-list component directly. There's an option to force it to not collapse